README.md

```diff
@@ -122,6 +121,27 @@ If you don't want to display comments for a particular post you can disable them
 
 --
 
+### Social networks
+
+You can add links to the accounts you have on other sites, with respective icon, by adding one or more of the following options in your config:
+
+```yaml
+twitter_username: jekyllrb
+github_username:  jekyll
+dribbble_username: dribbble
+facebook_username: fb
+flickr_username: flickr
+instagram_username: ig
+linkedin_username: linkedin
+pinterest_username: pinterest
+stackoverflow_username: stackoverflow
```

Comment:
Agreed, I'll try to improve them.

I forgot that Stack Overflow was like this.  Unfortunately, the premise of these social links is that the username doubles as both the link and the link text.  :-(

We could of course minimize to this:
```yaml
stackover_username: 111111/jekyll
```
But the link text would still be **111111/jekyll** unless it was filtered out.  Maybe punt on Stackoverflow for now?  See if people actually want it?  Stack Overflow have their own flair badges anyway, see: https://stackoverflow.com/users/flair